kubernetes/kserve/kserve_wrapper/TorchserveModel.py

```diff
@@ -1,23 +1,19 @@
 """ The torchserve side inference end-points request are handled to
     return a KServe side response """
-import json
 import logging
 import pathlib
-from typing import Dict
 
 import kserve
-import tornado.web
+from kserve.errors import ModelMissingError
 from kserve.model import Model as Model
-from kserve.model import ModelMissingError
 
 logging.basicConfig(level=kserve.constants.KSERVE_LOGLEVEL)
 
+PREDICTOR_URL_FORMAT = PREDICTOR_V2_URL_FORMAT = "http://{0}/predictions/{1}"
+EXPLAINER_URL_FORMAT = EXPLAINER_V2_URL_FORMAT = "http://{0}/explanations/{1}"
```

Comment:
Does we need support backward compatible (ie. v1) at here?

Reply:
yes, there is no plan for deprecating v1 format in near future.